docs/components/best-practices/architecture/sizing-your-environment.md

```diff
@@ -124,7 +124,9 @@ Spinning up a Camunda Platform 8 Cluster means you run multiple components that
 
 All components are clustered to provide high-availability, fault-tolerance and resiliency.
 
-Zeebe scales horizontally by adding more cluster nodes (pods). This is limited by the so-called partition size of a Zeebe cluster, as the work within one partition cannot be parallelized by design. Hence, you need to define enough partitions to utilize your cluster or to have some buffer if your load increases later on. The number of partitions cannot be changed after the cluster was initially provisioned (at least not yet), so elastic scalability of cluster nodes is (not yet) possible.
+Zeebe scales horizontally by adding more cluster nodes (pods). This is **limited by the [number of partitions](/docs/components/zeebe/technical-concepts/partitions/)** configured for a Zeebe cluster, as the work within one partition cannot be parallelized by design. Hence, you need to define enough partitions to utilize your hardware. The **number of partitions cannot be changed after the cluster was initially provisioned** (at least not yet), elastic scalability of partitions is (not yet) possible.
+
+If you foresee that the load will increase over time, you should prepare by configuring more partitions than you need just now, as this will be your buffer if your load increases later on. For examle, you could multiply the number of paritions you need for your current load by four to add buffer, which typically has a neglectable impact on performance.
```

Comment:
Small typos here -- would rewrite as:

**"If you anticipate the load increasing over time, prepare by configuring more partitions than you currently need as a buffer. For example, you could multiply the number of partitions you need for your current load by four to add a buffer. This typically has just a small impact on performance."**

Thoughts?

Reply:
Works 👍